third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -374,34 +387,18 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
         &mut self,
         node_id: NodeId,
         ty: &Type,
-        reported_types: &mut BTreeSet<Type>,
         reported_vars: &mut BTreeSet<u32>,
```

Comment:
It would be useful to state in the comment above that
```
/// Free type variables found are added to `reported_vars`.
```

Reply:
Done